src/components/backend-ai-login.ts

```diff
@@ -1979,6 +1979,25 @@ export default class BackendAILogin extends BackendAIPage {
       : password.setAttribute('type', 'password');
   }
 
+  private _validateInput() {
+    // Email validation pattern
+    const emailPattern = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
```

Comment:
Please use simpler client-side validation programmatically because it is difficult to validate perfectly.
/^[^\s@]+@[^\s@]+\.[^\s@]+$/;